src/Sylius/Bundle/GridBundle/Form/Type/Filter/EntityFilterType.php

```diff
@@ -33,10 +33,6 @@ public function configureOptions(OptionsResolver $resolver)
                 'placeholder' => 'sylius.ui.all',
             ])
             ->setRequired('class')
```

Comment:
It can be removed as well (along with `class` in `setDefaults`) as it's already defined in parent type.